docs/source/guide/ddd-1-intro.myst

```diff
@@ -12,3 +12,115 @@ kernelspec:
 ---
 
 # How do I use DDD?
+
+Digital Dynamical Decoupling (DDD) is an error mitigation technique in which
+sequences of gates are applied to slack windows, i.e. single-qubit idle
+windows, in a quantum circuit. Mitiq provides standard built-in rules that can
+be directly imported from `mitiq.ddd.rules`. 
+For more discussion of the theory of DDD, see the section [What is the theory
+behind DDD?](ddd-5-theory.myst).
+
+
+As with all techniques, DDD is compatible with any frontend supported by Mitiq:
+
+```{code-cell} ipython3
+import mitiq 
+
+mitiq.SUPPORTED_PROGRAM_TYPES.keys()
+```
+
+
+## Problem setup
+We first define the circuit of interest. In this example, the circuit has a
+slack window with a slack length of 4.
+
+```{code-cell} ipython3
+from cirq import LineQubit, Circuit, rx, rz, CNOT
+a, b = LineQubit.range(2)
+circuit = Circuit(
+    rx(0.1).on(a),
+    rz(0.4).on(a),
+    rx(-0.72).on(a),
+    rz(0.2).on(a),
+    rx(-0.1).on(b),
+    CNOT.on(a, b),
+    rz(-0.23).on(a),
+    CNOT.on(b, a),
+    rx(-0.112).on(a),
+    rx(0.1).on(a),
+    rz(0.4).on(a),
+    rx(-0.72).on(a),
+    rz(0.2).on(a),
+    rx(-0.1).on(b),
+    CNOT.on(a, b),
+    rz(-0.23).on(a),
+    CNOT.on(b, a),
+    rx(-0.112).on(a),
+    )
```

Comment:
```suggestion
circuit = Circuit(
    rx(0.1).on(a),
    rx(0.1).on(a),
    rz(0.4).on(a),
    rx(-0.72).on(a),
    rz(0.2).on(a),
    rx(-0.8).on(b),
    CNOT.on(a, b),
)
```
Do you have a particular reason for the choice of the circuit? If not, I would suggest something simpler as the suggested one (it seems to work).

Reply:
I wanted to make sure the circuit depth was sufficient to show an improvement with DDD (I saw some inconsistency when running locally), but maybe it's better to just use a higher noise level.